### DISCUSSION TITLE
Add section dividers

### DISCUSSION THREAD

**Discussion by LivInTheLookingGlass** [iSoron/uhabits#1028] [Feature Requests] [answered]
As a current workaround, I can just have a blank habit as a section label, but it would be nice if this was officially supported by the app. That way I can sort things into headings like "exercise" and "diet" and "moral", among others

**Comment by snippet78** [comment]
I color code my 'sections' to make them easy to see and save space on the screen.

**Comment by hiqua** [comment, marked as answer]
See https://github.com/iSoron/uhabits/discussions/495
